src/components/search.js

```diff
@@ -39,11 +40,20 @@ export default class Search extends Component {
         render={data => {
           return (
             <div>
-              <input
-                type="text"
+              <TextField
+                className={'searchField'}
```

Comment:
Where is the searchField class defined?

Reply:
it' always good to have a css class for element, to style it later

Reply:
I thought it is a predefined class. Ok, so currently, it has not attributes.